### PR TITLE
Gnome terminal -> 3.41.0

### PR DESCRIPTION
**Pull request by satmandu**
Works properly:
- [x] x86_64

Builds properly:
- [x] x86_64
- [x] armv7l
- [ ] i686

### CODE REVIEW

File: packages/gnome_terminal.rb [hunk 1]
```diff
@@ -3,22 +3,22 @@
...
   homepage 'https://wiki.gnome.org/Apps/Terminal'
-  @_ver = '3.41.0-0f6f'
+  @_ver = '3.41.0'
   version @_ver
   license 'GPL-3+'
-  compatibility 'x86_64 aarch64 armv7l'
+  compatibility 'all'
   source_url 'https://gitlab.gnome.org/GNOME/gnome-terminal.git'
-  git_hashtag '0f6fb3781c2f5efe3306ebe6f41a7e4ba71880db'
+  git_hashtag '3.41.0'
```
Comment (supechicken) [inline packages/gnome_terminal.rb:11]: Do you mean `git_branch '3.41.0'`?

File: packages/gnome_terminal.rb [hunk 2]
```diff
@@ -3,22 +3,22 @@
...
 
   binary_url({
     aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/gnome_terminal/3.41.0-0f6f_armv7l/gnome_terminal-3.41.0-0f6f-chromeos-armv7l.tpxz',
      armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/gnome_terminal/3.41.0-0f6f_armv7l/gnome_terminal-3.41.0-0f6f-chromeos-armv7l.tpxz',
```
Comment (supechicken) [inline packages/gnome_terminal.rb:15]: New binary for `armv7l`?

File: packages/gnome_terminal.rb (hunk 1 above)
Comment (satmandu) [reply packages/gnome_terminal.rb:11]: No. That is a tag.

File: packages/vte.rb [hunk 3]
```diff
@@ -26,16 +26,6 @@ class Vte < Package
...
-  def self.patch
-    system "sed -i 's/gtk+-4.0/gtk4/g' meson.build"
-
-    # i686 headers are too old to have TIOCGPTPEER
-    case ARCH
-    when 'i686'
-      system "sed -i 's/#ifdef __linux__/#ifdef TIOCGPTPEER/' src/pty.cc"
-    end
-  end
-
```
Comment (uberhacker) [inline packages/vte.rb:38]: Does this patch removal require new binaries?

File: packages/gnome_terminal.rb (hunk 2 above)
Comment (satmandu) [reply packages/gnome_terminal.rb]: Done.

File: packages/vte.rb (hunk 3 above)
Comment (satmandu) [reply packages/vte.rb:38]: I don't think we should be making i686 builds for this, so no.

File: packages/gnome_terminal.rb [hunk 4]
```diff
@@ -3,22 +3,22 @@
 class Gnome_terminal < Package
   description 'The GNOME Terminal Emulator'
   homepage 'https://wiki.gnome.org/Apps/Terminal'
-  @_ver = '3.41.0-0f6f'
+  @_ver = '3.41.0'
   version @_ver
   license 'GPL-3+'
-  compatibility 'x86_64 aarch64 armv7l'
+  compatibility 'all'
```
[the remark this answers is not in the document]
Comment (satmandu) [reply packages/gnome_terminal.rb:9]: Same argument for leaving compatibility as all as in https://github.com/skycocker/chromebrew/pull/6132#discussion_r700453973

**Pull request merged**